site/index.html

```diff
@@ -0,0 +1,158 @@
+  <title>Lab 2 - Static Site</title>
+
+  <link rel="icon" type="image/png" href="/favicon-96x96.png" sizes="96x96" />
+  <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+  <link rel="shortcut icon" href="/favicon.ico" />
+  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+  <link rel="manifest" href="/site.webmanifest" />
+</head>
+<body>
+
+<div class="width-wrap">
+  <header id="top">
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="links.html">Links</a></li>
+        <li><a href="glossary.html">Glossary</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+
+    <section id="bio">
+      <h1>Ball is Life</h1>
+      <p>Basketball is a dynamic team sport where two teams of five players compete on a rectangular court to score points by shooting a ball through a hoop.<br>
+        The goal is to score more points than the opposing team while preventing them from scoring. <br>
+        The game is fast-paced and involves a lot of running, passing, and shooting.</p>
+    </section>
+
+
+    <table>
+
+      <caption>Table 1. Current Courses</caption>
+      <thead>
+      <!--<tr><td colspan ="4"> Courses</td></tr>-->
+        <tr><th colspan="4">Courses</th></tr>
+      <tr>
+        <td colspan="1">Term</td>
+        <td colspan="1">Code</td>
+        <td colspan="1">Name</td>
+        <td colspan="1">hours</td>
+      </tr>
+
+      </thead>
+
+      <thead>
+      <tr><td rowspan="8">25W</td><td>CSD 228</td><td>Mobile Application</td> <td>4</td></tr>
+      <tr><td>CSD 113</td><td>Web Development</td><td>4</td></tr>
+      <tr><td>CSD 442</td><td>Intro to Programming</td><td>4</td></tr>
+      <tr><td>CSD 122</td><td>Communication</td><td>4</td></tr>
+      <tr><td>CSD 442</td><td>Mathematics</td><td>4</td></tr>
+      <tr><td>CSD 442</td><td>Envy Tools</td><td>4</td></tr>
+      <tr><td>CSD 442</td><td>Networking</td><td>4</td></tr>
+      <tr><td>CSD 442</td><td>Academic Transition</td><td>4</td></tr>
+      </thead>
+
+      <thead>
+      <tr><td rowspan="3">25S</td><td>CSD 228</td><td>Mobile Application</td> <td>4</td></tr>
+      <tr><td>CSD 113</td><td>Web Development</td><td>4</td></tr>
+      <tr><td>CSD 442</td><td>Intro to Programming</td><td>4</td></tr>
+      </thead>
+
+
+
+    </table>
+
+    <section id="my-favourite-things">
+      <h2>My Favourite Things</h2>
+      <a href="https://www.freeiconspng.com/img/26259">
+        <img src="image/basketball.png" alt="Basketball Logo">
+      </a>
+
+      <section id="thumbnails-and-alternative-sizes">
+        <a href="image/myself.jpg">
+          <picture>
+            <source media="(min-width: 501px)" srcset="image/thumbnail-300.jpg">
+            <source media="(max-width: 500px)" srcset="image/thumbnail-150.jpg">
+            <img src="image/thumbnail-300.jpg" alt="Myself thumbnail">
+          </picture>
+        </a>
+      </section>
```

Comment:
> 4. `Lab 3 > Thumbnails and Alternative Sizes > img uses sizes attribute`
> 5. `Lab 3 > Thumbnails and Alternative Sizes > img srcset is setup correctly`
> 6. `Lab 3 > Thumbnails and Alternative Sizes > displays the 150px thumbnail when the viewport is under 500px`
> 7. `Lab 3 > Thumbnails and Alternative Sizes > displays the 300px thumbnail when the viewport is over 500px`
> 8. `Lab 3 > Thumbnails and Alternative Sizes > click on the displayed thumbnail opens the full image`
```suggestion
      <section id="thumbnails-and-alternative-sizes">
        <a href="image/myself.jpg" target="_blank">
          <picture>
            <img src="image/thumbnail-300.jpg" alt="Myself thumbnail" sizes="(max-width:500px) 150px, 300px" srcset="image/thumbnail-150.jpg 150w, image/thumbnail-300.jpg 300w">
          </picture>
        </a>
      </section>
```

> 9. `Lab 3 > Thumbnails and Alternative Sizes > alt attribute is supplied to the images`

Overrridden, unexpected nesting